txpipe/twopoint.py

```diff
@@ -596,12 +659,17 @@ class TXTwoPointPlots(PipelineStage):
     ]
     outputs = [
         ('shear_xi', PNGFile),
+        ('shear_xi_err', PNGFile),
         ('shearDensity_xi', PNGFile),
+        ('shearDensity_xi_err', PNGFile),
         ('density_xi', PNGFile),
+        ('density_xi_err', PNGFile),
     ]
 
     config_options = {
-
+        'plot_compare_errors': True,
```

Comment:
There's no need to make this optional, unless I'm missing something?  This will be a useful plot always.

Reply:
True, I removed the option.